docs/features/loadmanagement.mdx

```diff
@@ -0,0 +1,116 @@
+---
+sidebar_position: 9
+---
+
+# Lastmanagement
+
+:::warning Experimentell
+Lastmanagement befindet sich gerade noch in der Erprobungsphase.
+:::
+
+Lastmanagement ermöglicht es, die Leistung von Ladestationen zu steuern, um die Gesamtlast zu begrenzen.
+Hierfür können Ladepunkte in Gruppen zusammengefasst werden.
+Für jede Gruppe (`circuit`) kann eine maximale Stromstärke (`maxCurrent`) und/oder Leistung (`maxPower`) konfiguriert werden.
+Das System arbeitet hierarchisch, d.h. Gruppen können Teil einer übergeordneten Gruppe sein.
```

Comment:
Es muss *genau* eine übergeordnete Gruppe geben. Diese beschreibt den Netzanschluss.

Reply:
Zu prüfen: @premultiply 

Der übergeordneten Gruppe können keine Ladepunkte zugeordnet werden.

Reply:
> Der übergeordneten Gruppe können keine Ladepunkte zugeordnet werden.

Das heißt Gruppen (ich bleib mal bei dem Wort) können entweder Untergruppen haben oder(!) Ladepunkte aber niemals beides?

Reply:
Doch.
Immer an das Leitungsnetz denken.
Ein Circuit definiert quasi was über die Zuleitung einer Verteilung angeschlossen ist.

Bei der Hauptverteilung ist es die Leitung zum Netzbetreiber, bei einer Unterverteilung die Leitung zur Hauptverteilung (site-Circuit) oder einer andere übergeordneten Unterverteilung.

Reply:
Ok, ich habe jetzt dem Beispiel "Beispiel: Verschachtelte Gruppen" noch mal eine Wärmepumpe am `main` hinzugefügt um das zu verdeutlichen.

Reply:
@premultiply unter welchen Bedingungen kann eine Ladepunkt einem Circuit zugeordnet sein im Falle root circuit oder untergeordneter Circuit? Nach meinem Verständnis immer, aber nicht dem root circuit?

Reply:
Alle Ladepunkte sind immer im Root-Circuit es sei denn sie sind explizit in einem Sub-Circuit. In dem Fall ersetzt der Sub-Circuit die dort befindlichen Ladepunkte im übergeordneten Circuit.

Reply:
> Alle Ladepunkte sind immer (automatisch) im Root-Circuit es sei denn sie sind explizit in einem Sub-Circuit.

Find ich auch intuitiv aus Nutzersicht. Ich hab das im Text ergänzt.